src/components/MessagesList/MessagesGroup/MessagesSystemGroup.vue

```diff
@@ -198,41 +203,52 @@ export default {
 			return ''
 		},
 
+		updateCollapsedState() {
+			for (const group of this.messagesGroupedBySystemMessage) {
```

Comment:
I think it would be better to make it without mutations of reactive data. Could also try to do it with `Array.reduce()`:

```suggestion
		updateCollapsedState(groups) {
			return groups.reduce((acc, value) => cb, [])
```

Reply:
the data is reassigned in that case, I am just updating.